codegen/src/main/kotlin/tools/samt/codegen/http/HttpTransport.kt

```diff
@@ -0,0 +1,228 @@
+package tools.samt.codegen.http
+
+import tools.samt.api.plugin.TransportConfiguration
+import tools.samt.api.plugin.TransportConfigurationParser
+import tools.samt.api.plugin.TransportConfigurationParserParams
+import tools.samt.api.plugin.asEnum
+
+object HttpTransportConfigurationParser : TransportConfigurationParser {
+    override val transportName: String
+        get() = "http"
+
+    override fun default(): HttpTransportConfiguration = HttpTransportConfiguration(
+        serializationMode = HttpTransportConfiguration.SerializationMode.Json,
+        services = emptyList(),
+    )
+
+    private val isValidRegex = Regex("""\w+\s+\S+(\s+\{.*? in \S+})*""")
+    private val methodEndpointRegex = Regex("""(\w+)\s+(\S+)(.*)""")
+    private val parameterRegex = Regex("""\{(.*?) in (\S+)}""")
```

Comment:
will this regex work if there is extra whitespace arond the "in"?

Reply:
it will not, I've adjusted in accordingly